ios/OktaSdkBridge/OktaSdkBridge.swift

```diff
@@ -40,7 +40,7 @@ protocol StateManagerProtocol: AnyObject {
     var accessToken: String? { get }
     var idToken: String? { get }
     var refreshToken: String? { get }
-
+    
```

Comment:
Looks like there are quite a few unrelated white space changes.

Reply:
That's Xcode alignment when it aligns a file. Previous alignment was manual or custom.